From 6a053460f1ff003855bf6ffa9527f56063cd7bec Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Thu, 18 Sep 2014 14:14:28 -0400
Subject: [scsi] megaraid_sas: Driver version update (06.805.06.00-rc1)

Message-id: <1411049668-28841-29-git-send-email-thenzl@redhat.com>
Patchwork-id: 96170
O-Subject: [RHEL7.1 e-stor PATCH 28/28] megaraid_sas : Driver version update (06.805.06.00-rc1)
Bugzilla: 1088523
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

Driver version upgrade patch.

scsi-queue/drivers-for-3.18 commit 0d9d8b9ff4453e4816b22bf729256feb6b38e0ec

Signed-off-by: Sumit Saxena <sumit.saxena@avagotech.com>
Signed-off-by: Kashyap Desai <kashyap.desai@avagotech.com>
Reviewed-by: Tomas Henzl <thenzl@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/megaraid/megaraid_sas.h b/drivers/scsi/megaraid/megaraid_sas.h
index f99db18..a49914d 100644
--- a/drivers/scsi/megaraid/megaraid_sas.h
+++ b/drivers/scsi/megaraid/megaraid_sas.h
@@ -33,9 +33,9 @@
 /*
  * MegaRAID SAS Driver meta data
  */
-#define MEGASAS_VERSION    "06.803.02.00-rc1"
-#define MEGASAS_RELDATE    "Jun. 19, 2014"
-#define MEGASAS_EXT_VERSION   "Thu. Jun. 19 17:00:00 PDT 2014"
+#define MEGASAS_VERSION    "06.805.06.00-rc1"
+#define MEGASAS_RELDATE    "Sep. 4, 2014"
+#define MEGASAS_EXT_VERSION   "Thu. Sep. 4 17:00:00 PDT 2014"
 
 /*
  * Device IDs
diff --git a/drivers/scsi/megaraid/megaraid_sas_base.c b/drivers/scsi/megaraid/megaraid_sas_base.c
index 3cb2abe..4e6768d 100644
--- a/drivers/scsi/megaraid/megaraid_sas_base.c
+++ b/drivers/scsi/megaraid/megaraid_sas_base.c
@@ -18,7 +18,7 @@
  *  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
  *
  *  FILE: megaraid_sas_base.c
- *  Version : 06.803.02.00-rc1
+ *  Version : 06.805.06.00-rc1
  *
  *  Authors: LSI Corporation
  *           Sreenivas Bagalkote
-- 
1.7.1